### ISSUE TITLE
🐛 Bug Report: Appwrite-CLI :   appwrite deploy collection Failed with more then 25 Columns

### ISSUE THREAD

**Issue by punitdiwan** [appwrite/appwrite#6228]
### 👟 Reproduction steps

When I executed appwrite cli command to deploy the collection and selected a collection that had 33 fields, the command took a long time and returned an error.

### 👍 Expected behavior

It should deploy smoothly create the collection in the Appwrite database and start processing the next collection if it exists in **appwrite.json** file. .

### 👎 Actual Behavior

It created a collection but did not assign permissions mentioned in **appwrite.json** file. The Command stuck for some time and then it returns after a long time showing the following Error on both cmd and PowerShell:

**On Power Shell**
![image](https://github.com/appwrite/appwrite/assets/49054503/324bd174-3a77-42a8-90ec-a88262fb56df)

**On Cmd**
![image](https://github.com/appwrite/appwrite/assets/49054503/366454ff-f770-4a34-b02b-c841ef09300f)

### 🎲 Appwrite version

Version 1.4.x

### 💻 Operating system

Windows

### 🧱 Your Environment

Windies: 10
Appwrite: 1.4.1
Appwrite-Cli: 2.0.2
CLI: Power Shell and CMD  on window 10

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by stnguyen90** [comment]
@punitdiwan, thanks for creating this issue! 🙏🏼 It looks like you're using version 2.0.2 of the Appwrite CLI. Would you please try using version 4.0.0 instead? If the problem is still occurring, would you please provide some info about why the attributes failed to create? You may need to check the docker logs for the `appwrite-worker-databases` container.

**Comment by punitdiwan** [comment]
@stnguyen90 I am thankful for your prompt response. 

 As you suggested, I have upgraded the Appwrite CLI to 4.0.0.  I am very happy to share that the problem has been fixed and not I am able to create collections with large fields. The large number of fields in the collections which I have is 40. It was smoothly created when I used `appwrite deploy collection` with appwrite-cli 4.0.0.

As this issue is resolved, it is good to close. So, I am closing the issue.
